pkg/cmd/pr/create/regex_writer.go

```diff
@@ -0,0 +1,62 @@
+package create
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"regexp"
+)
+
+func NewRegexWriter(out io.Writer, regexp *regexp.Regexp, repl string) io.Writer {
+	return &RegexWriter{out: out, regexp: *regexp, repl: repl}
+}
+
+type RegexWriter struct {
+	out    io.Writer
+	regexp regexp.Regexp
+	repl   string
+}
+
+func (s RegexWriter) Write(data []byte) (int, error) {
+	filtered := []byte{}
```

Comment:
I think you will have a problem if you get `Write()` calls each with incomplete data for a line (e.g. one `Write()` call ending in the first half of the message you want to filter, and then the next starting halfway through the line).